symplyphysics/laws/chemistry/cross_section_of_interaction_in_recharge_model.py

```diff
@@ -0,0 +1,67 @@
+from sympy import Eq, solve, pi, log, sqrt
+from sympy.physics.units import elementary_charge, boltzmann_constant
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    clone_symbol,
+    symbols
+)
+
+# Description
+## The effective cross section is a physical quantity characterizing the probability of transition of a system of
+## two interacting particles to a certain final state, a quantitative characteristic of the acts of collision of
+## particles of a stream hitting a target with target particles. The effective cross-section has the dimension of the area.
+
+## Law is: g = pi * a0^2 * (Uh / Ui) * (ln(sqrt(3 * k * T / m) * (g * P * m / (2 * k * T * q * E)) * sqrt(Ui / Uh)))^2, where
+## g - the cross-sectional area of the interaction of particles,
+## Ui - the ionization energy of atoms,
+## T - temperature,
+## m - mass of atom,
+## P - pressure
+## E - electric intensity,
+## Uh - hydrogen ionization energy,
+## a0 - bohr radius,
+## k - boltzmann constant,
+## q - elementary charge.
+
+cross_sectional_area_of_interaction = Symbol("cross_sectional_area_of_interaction", units.area)
+
+ionization_energy = Symbol("ionization_energy", units.energy)
+mass_of_atom = clone_symbol(symbols.basic.mass, "mass_of_atom")
+pressure = Symbol("pressure", units.pressure)
+temperature = clone_symbol(symbols.thermodynamics.temperature, "temperature")
+electric_intensity = Symbol("electric_intensity", units.voltage / units.length)
+
+bohr_radius = Quantity(0.529e-10 * units.meter)
+hydrogen_ionization_energy = Quantity(13.6 * units.electronvolt)
```

Comment:
Move to global quantities.

Reply:
> Move to global quantities.

What does it mean?

Reply:
We put quantities here: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/quantities/__init__.py